config/objects/generic.json

```diff
@@ -549,7 +559,17 @@
 				"rmg" : {
 					"value"		: 2500,
 					"rarity"	: 20
-				}
+				},
+				"modes" : ["resource-skill"],
+				"title" : "core.genrltxt.602",
+				"speech" : "core.genrltxt.603",
+				"offer": 
+				[
+					{ "noneOf" : ["necromancy"] },
+					{ "noneOf" : ["necromancy"] },
+					{ "noneOf" : ["necromancy"] },
+					{ "noneOf" : ["necromancy"] }
```

Comment:
I think this is hota change, not H3 mechanics.

Not against this change though - since this is always a dead slot:
- non-Necropolis heroes can't learn Necromancy due to having it banned for their class
- Necropolis heroes can't learn Necromancy here since they already have it as staring skills

Reply:
I took it from FizMig 2.5, but yes it includes hota. I'm not an expert in h3 mechanics, so can remove if so. As you mentioned, it's dead slot otherwise

Reply:
@Nordsoft91 In FizMig hota changes should shown with hota icon. If this is shown with hota icon - it is hota change.

Reply:
As I said - not against this change. If nobody else objects - go ahead.